integration/test_flow_integration.py

```diff
@@ -636,7 +671,10 @@ def test_inner_join_by_key(setup_kv_teardown_test):
 
 
 def test_write_table_specific_columns(setup_teardown_test):
-    table = Table(setup_teardown_test.table_name, setup_teardown_test.driver())
+    if setup_teardown_test.driver_name == "SQLDriver":
+        pytest.skip("test_write_table_specific_columns not testing over SQLDriver")
```

Comment:
pytest has "skipif" for these cases - please replace

Reply:
I cant use skipif inside function